sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -330,10 +330,10 @@
 public class CaseFacadeEjb extends AbstractCoreFacadeEjb<Case, CaseDataDto, CaseIndexDto, CaseReferenceDto, CaseService, CaseCriteria>
 		implements CaseFacade {
 
-	private static final int ARCHIVE_BATCH_SIZE = 1000;
-
 	private final Logger logger = LoggerFactory.getLogger(getClass());
 
+	@EJB
+	private CaseService caseService;
```

Comment:
According to Barna's comment, probably also not needed because we can directly access the protected service field?